### ISSUE TITLE
Bug: Keyboard Navigation doesn't work when moving into an external link

### ISSUE THREAD

**Issue by noahlitvin** [wevm/vocs#236]
Try "Shift →" here: https://docs.foil.xyz/price-glossary - I get a 404 at `https://docs.foil.xyz/price-glossary/https://usecannon.com/packages/foil`

Code: https://github.com/foilxyz/foil/tree/main/packages/docs

Love this project, ty for the work on it 🙏

**Comment by dschlabach** [comment]
I have a fix PR for this here https://github.com/wevm/vocs/pull/226
